project/plugins.sbt

```diff
@@ -24,3 +24,5 @@ addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.0.4")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
 
 addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.5.2")
+
+addSbtPlugin("com.sksamuel.scapegoat" %% "sbt-scapegoat" % "1.0.4")
```

Comment:
ok then I was just mislead by you adding a scapegoat plugin here - I thought you were planning to use it. So maybe we should remove this line and introduce scapegoat in a different PR?

Reply:
agree!